### DISCUSSION TITLE
Help With Error 404

### DISCUSSION THREAD

**Discussion by JeffersonMora01** [vercel/community#7360] [Help] [answered]
### Summary

Hello, I have a problem when trying to publish my application, it gives me a 404 error that I have been trying to correct for some time and I have not been able to do it.

### Example

The id error is iad1::bdnz7-1720034105088-e7e37b2b5313

### Steps to Reproduce

![image](https://github.com/vercel/community/assets/130717898/c9347ed9-8f5e-4bb2-83db-bbda35679bb6)

**Comment by amyegan** [comment, marked as answer]
Hey @JeffersonMora01. There's not enough info here for me to be able to recommend any specific solution. But I've previously shared some ideas in https://github.com/vercel/community/discussions/6980#discussioncomment-9705033 which should help.

If you give those ideas a try and still need help, please share a [minimal reproducible example](https://vercel.com/guides/creating-a-minimal-reproducible-example) so I can take a closer look 🙂

[a comment by an automated account is not shown]
